data/rahul152-ai.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Rahul Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "New in opens source",
```

Did you mean `open source`.
```suggestion
  "bio": "New to open source",
```